src/common/currentSigningMessage.ts

```diff
@@ -0,0 +1,14 @@
+export default function currentSigningMessage(currentRound: number) {
```

Comment:
+1 -- very nice. This will make it super easy to quickly modify the message with no fallout.